src/components/checkBoxSelection/checkBoxSelection.scss

```diff
@@ -36,17 +28,62 @@
   margin-top: 20px;
   display: none;
 }
-
 .card-settings {
-  height: 600px;
+  height: 350px;
+  margin-left: 159px;
   overflow-y: scroll;
-  background-color: #b5cfe1;
-  width: 350px;
-  padding-left: 20px;
-  box-shadow: 10px 10px 55px 0px black;
+  background-color:#b5cfe1 ;
+  width: 600px;
+  box-shadow: 10px 10px 55px 0px #000000;
 }
-
 .slider {
   margin-top: 10px;
   margin-bottom: 10px;
 }
+.opt {
+  margin-left: 350px;
+  padding-bottom: 800px;
+}
+.home-display-class {
+  display: flex;
+}
+.choose {
+  width: 50%;
+}
+.display {
+  width: 50%;
+}
+
+.optimize-button{
+  margin-top: 12%;
+  margin-top: 19%;
+  text-align: center;
+  display: inline-block;
+  justify-items: center;
+  width:100px;
+  height: 70px;
+  border-radius: 35px;
+  background-color: #b5cfe1;
+  align-items: center;
+}
+
+.download-Button {
+  text-align: center;
```

Comment:
does this do anything if you justify items center will it not apply create the desired effect already

Reply:
Please Do NOT resolve comments I have made

Reply:
have you tried removing the justify content or the text-align to see if the styling changes

Reply:
I removed most of unused decoration that is no longer relevant.

Reply:
please push your changes as they do not reflect